### PR TITLE
[ERRORS ON DEV][SWE-1635] Fixed errors on development - rewrote ind…

### PR DESCRIPTION
**Pull request by Anhelina4**
…exes and file-names #done

### CODE REVIEW

File: src/app/screens/AppIcon/screens.js [hunk 1]
```diff
@@ -1,7 +1,7 @@
-import AppIconAndroidScreen from './AppIconContent/AppIconAndroid/AppIconAndroidScreen'
-import AppIconGuideScreen from './AppIconContent/AppIconGuide/AppIconGuideScreen'
-import AppIconIntroScreen from './AppIconIntro/AppIconIntroScreen'
-import AppIconIosScreen from './AppIconContent/AppIconIos/AppIconIosScreen'
+import { default as AppIconAndroidScreen } from './AppIconContent/AppIconAndroid/AppIconAndroidScreen'
```
Comment (alexpassss) [inline src/app/screens/AppIcon/screens.js:1]: Bad import

File: src/app/screens/Color/screens.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
-import ColorIntroScreen from './ColorIntro/ColorIntroScreen'
+import { default as ColorIntroScreen } from './ColorIntro/ColorIntroScreen'
```
Comment (alexpassss) [inline src/app/screens/Color/screens.js:1]: Same here

File: src/app/screens/Elements/screens.js [hunk 3]
```diff
@@ -1,3 +1,3 @@
-import ElementsIntroScreen from './ElementsIntro/ElementsIntroScreen'
+import { default as ElementsIntroScreen } from './ElementsIntro/ElementsIntroScreen'
```
Comment (alexpassss) [inline src/app/screens/Elements/screens.js:1]: Same here

Comment (alexpassss) [reply src/app/screens/Elements/screens.js:1]: Strange file name

**Review by alexpassss — CHANGES_REQUESTED** [review]
Same mistakes across PR

File: src/app/screens/Elements/screens.js (hunk 3 above)
Comment (Anhelina4) [reply src/app/screens/Elements/screens.js:1]: What name should I use instead? I was consulting with Olena and Maks and they suggested that name. It is a file which contain all screens related to a topic(Typography, Imagery, App Icon) in correct order: Intro and then all the screen with content

**Review by alexpassss — APPROVED** [review]

**Pull request merged**